microprofile.ls/org.eclipse.lsp4mp.ls/src/main/java/org/eclipse/lsp4mp/utils/EnvUtils.java

```diff
@@ -0,0 +1,30 @@
+/*******************************************************************************
+* Copyright (c) 2022 Red Hat Inc. and others.
+*
+* This program and the accompanying materials are made available under the
+* terms of the Eclipse Public License v. 2.0 which is available at
+* http://www.eclipse.org/legal/epl-2.0, or the Apache License, Version 2.0
+* which is available at https://www.apache.org/licenses/LICENSE-2.0.
+*
+* SPDX-License-Identifier: EPL-2.0 OR Apache-2.0
+*
+* Contributors:
+*     Red Hat Inc. - initial API and implementation
+*******************************************************************************/
+package org.eclipse.lsp4mp.utils;
+
+/**
+ * Environment variable utility class
+ *
+ */
+public class EnvUtils {
+	/**
+	 * Return true if the variable is an ENV variable, and false otherwise
+	 *
+	 * @param variable the variable to check
+	 * @return true if the variable is an ENV variable, and false otherwise
+	 */
+	public static boolean isEnvVariable(String variable) {
+		return variable.toUpperCase().equals(variable);
```

Comment:
Why you don't use Characher.isUpperCase like I suggested in https://github.com/eclipse/lsp4mp/pull/247#discussion_r888985405 

`variable.toUpperCase()` will create a new instance of String, it is real shame to do that just to check that variable is uppercase.